packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -276,19 +277,30 @@ export default {
         async submitCheckout () {
             try {
                 const checkoutData = {
-                    mobileNumber: this.customer.mobileNumber
+                    customer: {
```

Comment:
This can potentially be pulled out into a mapper to keep this method trimmed down.

Reply:
Yeah, I think we should do that so the `submitCheckout` method is small and simple.

Reply:
@SachaZvetelman Do you mind if I do this in a follow-up PR?